src/test/java/automation/tests/ManagingGroupTest.java

```diff
@@ -0,0 +1,75 @@
+package automation.tests;
+
+import automation.utils.loaders.EnvironmentConfigLoader;
+import automation.utils.loaders.GroupPropLoader;
+import automation.utils.loaders.UserConfigLoader;
+import automation.pages.BrowsePage;
+import automation.pages.LoginPage;
+import automation.pages.NewGroupPage;
+import org.junit.jupiter.api.*;
+import org.openqa.selenium.WebDriver;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class ManagingGroupTest {
+
+    private static LoginPage loginPage;
+    private static WebDriver driver;
+    private static UserConfigLoader userConfLoader = new UserConfigLoader("user");
+    private static EnvironmentConfigLoader envConfLoader = new EnvironmentConfigLoader("environment");
+    private NewGroupPage newGroupPage;
+    private BrowsePage browsePage;
+    private GroupPropLoader groupPropLoader = new GroupPropLoader("group");
+
+    //TC01 - New group can be added
+    @Test
+    void checkIfNewGroupCanBeCreated() {
+        //given:
+        driver.get(envConfLoader.getNewGroupPanel());
+
+        //when:
+        String identifier = groupPropLoader.getGroupIdentifier();
+        String displayName = groupPropLoader.getGroupDisplayName();
+
+        newGroupPage = new NewGroupPage(driver);
+        newGroupPage.typeIdentifier(identifier);
+        newGroupPage.typeDisplayName(displayName);
+        newGroupPage.submitCreateGroupButton();
+
+        browsePage = new BrowsePage(driver, displayName, identifier);
+
+        driver.get(envConfLoader.getAdminBrowsePanel());
+        driver.navigate().refresh();
```

Comment:
What is this refresh for?

Reply:
Don't know why but when running my script, after clicking on 'Create User' button, page doesn't contain new group - so I'm refreshing it to get updated list of groups.